packages/api-clients/open-api/bffApi.yml

```diff
@@ -3773,7 +3773,7 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Problem"
-  /eservices/{eServiceId}/update:
```

Comment:
Here https://github.com/pagopa/interop-be-monorepo/pull/1273/files#diff-09c9a8d490c6f40553523dd2c43b8e1eea1de7e649804d4ba204466f46df2696R1027 I made some similar changes so this PR, in theory, should be already implemented in the "incaricato" branch.